llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2280,22 +2280,6 @@ MemoryDepChecker::isDependent(const MemAccessInfo &A, unsigned AIdx,
     return Dependence::Backward;
   }
 
-  // Positive distance bigger than max vectorization factor.
```

Comment:
The first part of the comment before the TODO still seems useful, although it may need rewording?

Reply:
Sure, we could write a fresh comment: what would you like it to say?